src/unum/unum.c

```diff
@@ -404,6 +412,10 @@ static void print_usage(int argc, char *argv[])
     printf(" --sysinfo-period <0-...>     - sysinfo reporting interval\n");
     printf("                                0: disable reporting\n");
     printf(" --dns-timeout <1-...>        - timeout in seconds for dns request\n");
+#ifdef FEATURE_GZIP
+    printf(" --gzip-requests <0-...>      - message compression threshold\n");
+    printf("                                0: turn-off message compression\n");
```

Comment:
this can be changed to  `0: compression disabled (default)` or `0: no compression (default)` to indicate the default setting.